src/ui/app/PageLayout.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ActivityStreamContainer from 'ui/activity-stream/ActivityStreamContainer';
+import NotificationListContainer from 'ui/activity-stream/NotificationListContainer';
+import VerticalMenuContainer from 'ui/internal-page/VerticalMenuContainer';
+import AboutInfoModal from 'ui/about/AboutInfoModal';
+import LicenseInfoModal from 'ui/license/LicenseInfoModal';
+
+const PageLayout = ({ className, children }) => (
+  <div
+    data-testid="page-layout"
+    className={[className, 'layout-pf-fixed'].join(' ').trim()}
```

Comment:
I think we don't need prop className, we can remove it

Reply:
done ✅